sources/Interop/Windows/um/winnt/ACCESS_REASON_TYPE.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/winnt.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+namespace TerraFX.Interop
+{
+    public enum ACCESS_REASON_TYPE
```

Comment:
This looks like it might be intended as a `Flags`?

Reply:
I wasn't aware that these situations weren't detected by ClangSharp. I'm guessing there are a lot of these, if not.

Reply:
I will look for additional examples for these and correct them as necessary.

Reply:
Correcting these in a future PR.